### DISCUSSION TITLE
2.6.0.9

### DISCUSSION THREAD

**Discussion by sjohnson1021** [AxionDrak/GameCube-Backup-Manager#81] [Announcements]
<img width="969" alt="Screenshot 2022-09-01 020218" src="https://user-images.githubusercontent.com/8071133/187852399-db4074a9-fc1c-4568-b872-2b37068f79f5.png">

"Check for Update" Will actually find this one. :-) 
### Alright. Let's Talk.

Some of the recent builds have been.. a gamble. And for that I apologize. I will invest time and energy into learning to work with Unit Tests. This way 1, standard functionality is basically guaranteed to be immediately noticed if its broken, We can generate more meaningful and precise data to use for optimization,  and there's less pressure/uncertainty when downloading an incremental release. Which means you guys get a faster program, that works more consistently, and there will be reports to verify it has been tested.

I'd like to thank all of you who have taken the time to test, and communicate here or in the forums. It is very helpful, and encouraging to see people still interested in a (t- ~14days) 21 year old console, and its games. Not to mention that you are willing to lend your time and effort to further the project.

I look forward to working with what/who's to come with the upcoming migration and rewrite.

Keep an eye out for 2.6.1.0 soon. There will be nothing major changed. as this version has essentially reached end of life. We are in the process of stripping it down to it's core, and rewriting it to be 🎶better, faster, .. stronger? no.. more stable, and more accessible. I can make no promises.. But MAUI apps can run on Android and iOS.. so.. .. ..... just saying. 🤔

## What's Changed
* Update frmAboutBox.ko.resx by @DDinghoya in https://github.com/AxionDrak/GameCube-Backup-Manager/pull/77
* Update frmMain.ko.resx by @DDinghoya in https://github.com/AxionDrak/GameCube-Backup-Manager/pull/78
* Fixed a lot, went through and did a lot of optimizations. by @sjohnson1021 in https://github.com/AxionDrak/GameCube-Backup-Manager/pull/76

**Full Changelog**: https://github.com/AxionDrak/GameCube-Backup-Manager/compare/2.0.0.0...2.6.0.9

<img width="1136" alt="Screenshot 2022-09-01 011311" src="https://user-images.githubusercontent.com/8071133/187847679-d4f4af7b-dd9f-4ab4-a4a3-a082616818a1.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/AxionDrak/GameCube-Backup-Manager/releases/tag/2.6.0.9'>2.6.0.9</a>.</em>

**Comment by Musicislife6984** [comment]
How can you enable to Keep only disk 1 folder and delete disk 2 folder? It is grayed out and will not let me choose it. I would like to not have to move files and rename the original disk 1 folder.

**Reply by sjohnson1021** [reply]
Currently I believe this functionality has been put off. For one it would mean possibly destroying data, possibly and even likely unintentionally.

This might be added into a future release onec we get to where we are going with v3.0

**Comment by dogchainx** [comment]
Great program so far.  The download covers for boxart doesn't seem to work on my system.  Any ideas?
